edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.DiagnosticsComponent/Uploader/LogAnalyticsMetricsUpload.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.Azure.Devices.Edge.Agent.MetricsCollector
+{
+    using System;
+    using System.Collections;
+    using System.Collections.Generic;
+    using System.Text;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    class LogAnalyticsMetricsUpload : IMetricsUpload
```

Comment:
Will EdgeAgent ever send to LogAnalytics directly? If not, then we should not add this.

Reply:
Eventually, we want to make this functionality available to customers. For now, it only goes to us. I'll remove it until then, since it's just copied from your sample.